### PR TITLE
Fix zarr private links for volume tracings and some minor improvements to private links UI

### LINKED ISSUE #6514: Copied zarr private links for volume annotation layers contain tracing id instead of name
the layer I got from the modal was http://localhost:9000/data/annotations/zarr/JBD2R1jHaAS10D5G/d7f00e6b-b1ee-4085-8f03-3f99e43ea5d3, should have been http://localhost:9000/data/annotations/zarr/JBD2R1jHaAS10D5G/mySegmentationLayer

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://fixzarrprivatelinksforvolumetracings.webknossos.xyz

### Steps to test:
- create some private links for an annotation and edit the expiration date to different dates to make sure the rendering makes sense

### Issues:
- fixes #6514
- contributes to #6474

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

**Review by daniel-wer — APPROVED** [review]
Nice improvements :+1: I added some minor comments.

File: frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx [hunk 1]
```diff
@@ -263,17 +270,22 @@ function ExpirationDate({ linkItem }: { linkItem: ZarrPrivateLink }) {
       </Tooltip>
     ) : null;
 
+  const expirationMoment = moment(linkItem.expirationDateTime);
   return (
     <span>
-      <FormattedDate timestamp={linkItem.expirationDateTime} />
+      <FormattedDate
+        format="YYYY-MM-DD"
+        tooltipFormat="YYYY-MM-DD HH:mm"
```
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx:278]: Maybe we just show the UTC expiration time in the UI as well. I think that is a fair default for a global audience.

File: frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx [hunk 2]
```diff
@@ -289,12 +301,27 @@ function ExpirationDate({ linkItem }: { linkItem: ZarrPrivateLink }) {
...
+  const hourDiff = expirationMoment.diff(now, "hours");
+
+  const duration =
+    hourDiff < 24
+      ? now.to(expirationMoment)
+      : // Expiration dates usually end at 23:59 UTC. If now == 1 day before the
+        // expiration date at 08:00, moment.to() would round the duration and
+        // render "2 days" which is confusing if the user selected (in 1 day).
+        // Therefore, we pin the time at each date to 23:59 UTC.
+        now.startOf("day").to(expirationMoment.startOf("day"));
```
[the remark this answers is not in the document]
Comment (philippotto) [reply frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx]: you are right, I changed it :+1:

File: frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (philippotto) [reply frontend/javascripts/oxalis/view/action-bar/private_links_view.tsx]: ok, I changed it so that the time is always displayed.

**Pull request merged**